derive dcattheme from info.json family

mikeAdamss opened the pull request:
this: https://github.com/GSS-Cogs/gss-utils/issues/169

rossbowen reviewed (commented):
Does `self.dataset.theme` get output anywhere? Wondering whether it needs to appear in `as_quads`. If you run this does it output the expected RDF?

mikeAdamss commented:
it adds it to the .trig file along with the other dataset metadata, as follows (look for `dcat:theme gdp:trade .`:

```
@prefix dcat: <http://www.w3.org/ns/dcat#> .
@prefix dct: <http://purl.org/dc/terms/> .
@prefix gdp: <http://gss-data.org.uk/def/gdp#> .
@prefix gov: <https://www.gov.uk/government/organisations/> .
@prefix ns1: <http://gss-data.org.uk/graph/gss_data/trade/> .
@prefix ns2: <http://xmlns.com/foaf/0.1/> .
@prefix ns3: <urn:x-rdflib:> .
@prefix pmd: <http://publishmydata.com/def/dataset#> .
@prefix pmdcat: <http://publishmydata.com/pmdcat#> .
@prefix qb: <http://purl.org/linked-data/cube#> .
@prefix rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#> .
@prefix rdfs: <http://www.w3.org/2000/01/rdf-schema#> .
@prefix spdx: <http://spdx.org/rdf/terms#> .
@prefix theme: <http://gss-data.org.uk/def/concept/statistics-authority-themes/> .
@prefix void: <http://rdfs.org/ns/void#> .
@prefix xml: <http://www.w3.org/XML/1998/namespace> .
@prefix xsd: <http://www.w3.org/2001/XMLSchema#> .

ns1:gss-utils-metadata {
    <http://gss-data.org.uk/catalog/datasets> a dcat:Catalog ;
        dcat:record <http://gss-data.org.uk/data/gss_data/trade/gss-utils-catalog-record> .

    <http://gss-data.org.uk/data/gss_data/trade/gss-utils#dataset> a pmdcat:DataCube .

    <http://gss-data.org.uk/data/gss_data/trade/gss-utils-catalog-entry> a pmdcat:Dataset ;
        rdfs:label "UK tariffs"@en ;
        gdp:family gdp:trade ;
        pmdcat:datasetContents <http://gss-data.org.uk/data/gss_data/trade/gss-utils#dataset> ;
        pmdcat:graph ns1:gss-utils ;
        dct:creator gov:department-for-international-trade ;
        dct:description "Use this service to check the UK Global Tariff that will apply to goods you import from 1 January 2021. You can also check the difference between what you pay now and what you’ll pay from 1 January 2021."@en ;
        dct:issued "2020-12-02"^^xsd:date ;
        dct:modified "2021-02-03T16:03:26.278579+00:00"^^xsd:dateTime ;
        dct:publisher gov:department-for-international-trade ;
        dct:title "UK tariffs"@en ;
        void:sparqlEndpoint <http://gss-data.org.uk/sparql> ;
        dcat:landingPage <https://www.gov.uk/guidance/uk-tariffs-from-1-january-2021> ;
        dcat:theme gdp:trade .

    <http://gss-data.org.uk/data/gss_data/trade/gss-utils-catalog-record> a dcat:CatalogRecord ;
        rdfs:label "UK tariffs Catalog Record"@en ;
        pmdcat:metadataGraph ns1:gss-utils-metadata ;
        dct:issued "2020-12-02"^^xsd:date ;
        dct:modified "2021-02-03T16:03:26.278579+00:00"^^xsd:dateTime ;
        ns2:primaryTopic <http://gss-data.org.uk/data/gss_data/trade/gss-utils-catalog-entry> .
}
```

should pul through from there into the RDF along with the other stuff here.
